bokehjs/src/lib/models/glyphs/webgl/base.ts

```diff
@@ -43,6 +44,34 @@ export abstract class BaseGLGlyph {
   }
 
   abstract draw(indices: number[], mainglyph: any, trans: Transform): void
+
+  // Return array from FloatBuffer, creating it if necessary.
+  protected get_buffer_array(float_buffer: FloatBuffer, length: number): Float32Array {
+    if (float_buffer == null || float_buffer.array.length != length)
```

Comment:
Sadly TypeScript allows to compare anything to `null` by design. Please make all nullable types actually nullable.